simpleflow/step/submittable.py

```diff
@@ -92,3 +92,9 @@ def fn_steps_done(steps_done):
             workflow.get_steps_done_activity(),
             FuncGroup(fn_steps_done),
             send_result=True))
+
+    def propagate_attribute(self, attr, val):
+        """
+        Propagate the attribute to the related activities.
+        """
+        self.activities.propagate_attribute(attr, val)
```

Comment:
Right, the name is a bit ambiguous. For now, I may add few words in docstring to clarify it.